dashboard/app/jobs.go

```diff
@@ -1664,6 +1662,61 @@ func relevantBackportJobs(c context.Context) (
 	return
 }
 
+func updateBackportCommits(c context.Context, ns string, commits []dashapi.Commit) error {
+	if len(commits) == 0 {
+		return nil
+	}
+	perTitle := map[string]dashapi.Commit{}
+	for _, commit := range commits {
+		perTitle[commit.Title] = commit
+	}
+	bugs, jobs, jobKeys, err := relevantBackportJobs(c)
+	if err != nil {
+		return err
```

Comment:
Will additional details help us to debug problems here in the future? (2 error driven returns in this function)